.github/workflows/check_codeowners.yml

```diff
@@ -3,11 +3,12 @@ name: Check CODEOWNERS Entries
 on:
   pull_request:
     branches:
-      - '*'
+      - master
     types:
       - opened
       - synchronize
       - reopened
+      - edited
```

Comment:
Edited will kick off the this github action if the PR summary is edited and not for code changes. "synchronize" will kick off this workflow when there are changes.
Do we want to execute this workflow when the author is updating the PR summary?

Reply:
We want the check to run if the base branch is changed and `edited` was the way we found to do that. So yeah 😞 it will run every time the description or title is updated as well, which isn't necessary. But without this, the CODEOWNERS checks are stuck in the Pending state. See the **Testing done** section for details. I figured the Pending state would cause more confusion (and maybe more support requests) than having it run on every `edited` trigger.

Reply:
Ah! The base branch change. Thanks!